src/ctia/bulk/routes.clj

```diff
@@ -45,25 +46,27 @@
   (GET "/" []
        :return (s/maybe Bulk)
        :summary "GET many entities at once"
-       :query-params [{actors          :- [Reference] []}
+       :query-params [{actors :- [Reference] []}
```

Comment:
I see parts on your changes where you align and others like here where you remove the alignement, how do you define what should be aligned or not?

Reply:
I generally prefer aligning things with `M-x clojure-align <C-SPC>` where that makes sense. Sometimes it gets wonky e.g. in let forms. Then I don't do that. I see that most of our code is not aligned, if my habit becomes a problem, let me know I will try not to commit any code that is aligned.
This particular piece I think got skewed because @ereteog  and I made commits on top of each other's.

Reply:
Oh, I was wrong. This piece is not aligned because clojure-align doesn't align them, they are not "related".